services/docker-compose-build.yml

```diff
@@ -7,6 +7,23 @@
 #
 version: "3.8"
 services:
+  service-integration:
```

Comment:
minor: I've only heard it called `ooil`, I think we might as well call it by its colloquial name to avoid confusion, if you want ;)